lib/app/app_base.nit

```diff
@@ -122,5 +122,12 @@ fun app: App do return once new App
 # However, this value can be used in log files and communications with servers.
 fun bound_platform: String do return "none"
 
+if "NIT_TESTING".environ == "true" then exit 0
+
+if bound_platform == "none" then
```

Comment:
`bound_platform` should not be used to decide the behavior of the software. ￼Class refinement provide a safer and a static solution to apply variations.